src/history/test/HistoryTestsUtils.cpp

```diff
@@ -452,6 +452,8 @@ void
 CatchupSimulation::generateRandomLedger(uint32_t version)
 {
     auto& lm = mApp.getLedgerManager();
+    auto& herder = static_cast<HerderImpl&>(mApp.getHerder());
```

Comment:
why not just add `makeStellarValue` to `Herder` (the public contract)? I don't see a downside and it avoids having to expose the "impl" version everywhere